### DISCUSSION TITLE
Problem with Bulk Action remove meetings

### DISCUSSION THREAD

**Discussion by brittbuhl** [code4recovery/12-step-meeting-list#1048] [Q&A] [answered]
Hello, my name is Britt and I am new to using TSML and have noticed an issue that I wanted to see if anyone else had noticed.  

I had entered our entire District meeting list and noticed some duplicates so I want to the meetings list and filtered the meetings by region to find the duplicate meetings and remove them.  I found the meetings I wanted to remove and checked them.  

Then I selected the bulk action "Move to Trash" option and clicked apply.  Once I did this, my entire meeting list disappeared from the website.

I then went to the trash area and found the meetings there, selected them all and clicked "restore".  The meetings then moved to a "drafts" category (not published) and, lo and behold, my entire meeting list re-appeared on the website.  I left the duplicate meetings in the "drafts" area and they don't appear on the website so - mission accomplished.  But, I noticed that error with the bulk action button and wanted to see if anyone else had noticed this?

Note:  when I remove a meeting individually (send it to the trash folder) - this doesn't cause my entire meeting list to disappear.  It seems that only using the bulk action button does that.

Anyone else notice this issue?  Any suggestions?

Thanks a lot!

**Comment by joshreisner** [comment, marked as answer]
hm, sorry that happened to you @brittbuhl ! i was just able to recreate your experience on our demo site. i think the main use-case works (check a few meetings, delete, refresh meetings page) but i encountered the problem when i specifically:

* used the filter to filter to a specific region
* checked the "select all" checkbox
* bulk actions > move to trash
* it says no meetings entered yet, but un-filtering shows that all the remaining meetings are still there
* go to meetings page, it says "No meetings were found matching the selected criteria."

i can restore the right meetings by editing and saving changes on a single remaining meeting, i don't need to actually move the deleted meetings from the trash, so i suspect we have a bug that just invalidates our local cache. 

i'll make a ticket to get it fixed! thanks for reporting the issue

**Reply by brittbuhl** [reply]
Thank you so much!

**Reply by joshreisner** [reply]
here's the ticket if you want to track our progress: https://github.com/code4recovery/12-step-meeting-list/issues/1049

for now if that happens again just edit an existing meeting and hit save changes and the meeting list should come back

**Reply by brittbuhl** [reply]
Yeah, that's great to know.  I can just edit a single meeting and the meeting list will reappear.

I was originally freaking out because I thought I had to re-enter my whole list - glad that didn't happen!
